templates/partners/become-a-partner.html

```diff
@@ -429,20 +440,11 @@ <h3 class="p-muted-heading">Partnering with Canonical gives you:</h3>
             </p>
             <div class="mktField">
               <button type="submit" class="p-button--positive mktoButton">Apply now</button>
-              <input type="hidden" name="formid" class="mktoField" value="1260">
-              <input type="hidden" name="formVid" class="mktoField" value="1260">
-              <input type="hidden" name="lpId" class="mktoField" value="1558">
-              <input type="hidden" name="subId" class="mktoField" value="30">
-              <input type="hidden" name="munchkinId" class="mktoField" value="066-EOV-335">
               <input type="hidden" name="lpurl" class="mktoField" value="https://pages.ubuntu.com/become-a-partner.html?cr={creative}&amp;kw={keyword}">
-              <input type="hidden" name="cr" class="mktoField" value="">
-              <input type="hidden" name="kw" class="mktoField" value="">
-              <input type="hidden" name="q" class="mktoField" value="">
             </div>
           </div>
         </fieldset>
         <input type="hidden" name="returnURL" value="https://canonical.com/partners/thank-you" />
-        <input type="hidden" name="retURL" value="https://canonical.com/partners/thank-you" />
```

Comment:
@caldav I removed these fields, they look like legacy stuff. But can you confirm that we don't need them anymore? Do we maybe need to replace it with `utm_source` fields?